package.json

```diff
@@ -107,6 +107,7 @@
   "resolutions": {
     "dns-over-http-resolver": "^2.1.1",
     "chai": "^4.3.10",
-    "loupe": "^2.3.6"
+    "loupe": "^2.3.6",
+    "vite": "^5.0.7"
```

Comment:
is this related to the PR description?

Reply:
Yes it's related. Used this change to fix few unmet dependencies yarn warnings while fixing and testing the yarn lint script.